reference/client-access/python-sdk.md

```diff
@@ -0,0 +1,287 @@
+---
+description: Technical reference for the libSQL Python client library used to access Turso databases, including sample code.
+keywords:
+  - turso
+  - libsql
+  - python
+  - client
+  - sdk
+  - library
+  - example
+---
+
+# Python SDK
+
+The Python SDK comes with APIs for both synchronous and asynchronous code.
+
+## Installation
+
+Add the [libsql_client module] to your project using `pip`:
+
+```bash
+$ pip install libsql_client
```

Comment:
The canonical name of the package is `libsql-client` (but `libsql_client` also works, pip seems to treat `_` the same as `-`).

Reply:
It seems odd to install one name then import from a slightly different name. I don't know what the convention is here, by my leaning is to suggest the same for both.

Reply:
The Python convention is to use `kebab-case` as a package name, even though you always need to import a package using `kebab_case`. Rust also uses this convention.

Reply:
Weird to me, but fair enough!